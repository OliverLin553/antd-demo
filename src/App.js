import React from 'react';
import Footer from './components/Footer'
import './App.css';

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <img src="https://create-react-app.dev/img/logo.svg" className="App-logo" alt="logo" />
        <p>
          Edit <code>src/App.js</code> and save to reload.
        </p>
        <a
          className="App-link"
          href="https://reactjs.org"
          target="_blank"
          rel="noopener noreferrer"
        >
          Learn React
        </a>
      </header>
      <Footer />
    </div>
  );
}

export default App;
