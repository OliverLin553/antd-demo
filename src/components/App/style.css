@import '~antd/dist/antd.css';

@charset "UTF-8";@font-face {
  font-family: ProximaNova;
  font-style: normal;
  font-weight: 300;
  font-display:fallback;src: local("Proxima Nova Light"),local("ProximaNova-light"),url('https://www.weddingwire.com//assets/fonts/ProximaNova-light.woff2') format("woff2"),url('https://www.weddingwire.com//assets/fonts/ProximaNova-light.woff') format("woff"),url('https://www.weddingwire.com//assets/fonts/ProximaNova-light.ttf') format("truetype")
}

@font-face {
  font-family: ProximaNova;
  font-style: normal;
  font-weight: 400;
  font-display:fallback;src: local("Proxima Nova Regular"),local("ProximaNova-regular"),url('https://www.weddingwire.com//assets/fonts/ProximaNova-regular.woff2') format("woff2"),url('https://www.weddingwire.com//assets/fonts/ProximaNova-regular.woff') format("woff"),url('https://www.weddingwire.com//assets/fonts/ProximaNova-regular.ttf') format("truetype")
}

@font-face {
  font-family: ProximaNova;
  font-style: normal;
  font-weight: 600;
  font-display:fallback;src: local("Proxima Nova Semibold"),local("ProximaNova-semibold"),url('https://www.weddingwire.com//assets/fonts/ProximaNova-semibold.woff2') format("woff2"),url('https://www.weddingwire.com//assets/fonts/ProximaNova-semibold.woff') format("woff"),url('https://www.weddingwire.com//assets/fonts/ProximaNova-semibold.ttf') format("truetype")
}

input {
  outline: 0;
}

html, body, div, span, applet, object, iframe,
h1, h2, h3, h4, h5, h6, p, blockquote, pre,
a, abbr, acronym, address, big, cite, code,
del, dfn, em, img, ins, kbd, q, s, samp,
small, strike, strong, sub, sup, tt, var,
b, u, i, center,
dl, dt, dd, ol, ul, li,
fieldset, form, label, legend,
table, caption, tbody, tfoot, thead, tr, th, td,
article, aside, canvas, details, embed,
figure, figcaption, footer, header, hgroup,
menu, nav, output, ruby, section, summary,
time, mark, audio, video {
  margin: 0;
  padding: 0;
  border: 0;
  font-size: 100%;
  font: inherit;
  vertical-align: baseline;
  font-family: ProximaNova,Helvetica,Arial,sans-serif;
}
/* HTML5 display-role reset for older browsers */
article, aside, details, figcaption, figure,
footer, header, hgroup, menu, nav, section {
  display: block;
}
body {
  line-height: 1;
}
ol, ul {
  list-style: none;
}
blockquote, q {
  quotes: none;
}
blockquote:before, blockquote:after,
q:before, q:after {
  content: '';
  content: none;
}
table {
  border-collapse: collapse;
  border-spacing: 0;
}
