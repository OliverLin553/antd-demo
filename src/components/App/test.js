import React from 'react';
import { shallow } from 'enzyme';
import App from './';

describe('UI test #home', () => {
  it('shoule have title', () => {
    const wrapper = shallow(<App />)
    expect(wrapper).toMatchSnapshot()
  })
})
