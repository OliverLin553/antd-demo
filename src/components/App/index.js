import React from 'react';
import { bindActionCreators } from "redux"
import Actions from '../../actions'
import Layout from '../Layout';
import Header from "../Header";
import Footer from '../Footer'
import './style.css';
import { connect } from "react-redux";

class App extends React.Component {
  componentDidMount() {
    const { actions } = this.props;

    actions.posts.fetch()
  }

  render() {
    const { children } = this.props;

    return (
      <div>
        <Header />
        <Layout>
          {children}
        </Layout>
        <Footer/>
      </div>
    );
  }
}

const mapDispatchToProps = dispatch => ({
  actions: {
    posts: bindActionCreators(Actions.posts, dispatch)
  }
})

export default connect(null, mapDispatchToProps)(App)
