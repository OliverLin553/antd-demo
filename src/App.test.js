import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('UI test #home', () => {
  it('shoule have title', () => {
    const wrapper = shallow(<App />)
    expect(wrapper).toMatchSnapshot()
  })
})
