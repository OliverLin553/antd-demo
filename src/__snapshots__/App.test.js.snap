// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI test #home shoule have title 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="https://create-react-app.dev/img/logo.svg"
    />
    <p>
      Edit 
      <code>
        src/App.js
      </code>
       and save to reload.
    </p>
    <a
      className="App-link"
      href="https://reactjs.org"
      rel="noopener noreferrer"
      target="_blank"
    >
      Learn React
    </a>
  </header>
  <Footer />
</div>
`;
